Test/WebTestCase.php

```diff
@@ -79,22 +80,38 @@ protected function tearDown()
     }
 
     /**
+     * @return ContainerInterface
+     */
+    public function getContainer()
+    {
+        return $this->container;
+    }
+
+    /**
+     * @return Client
+     */
+    public function getClient()
+    {
+        return $this->client;
+    }
+
+    /**
      * Save request output and show it in the browser
      * See http://giorgiocefaro.com/blog/test-symfony-and-automatically-open-the-browser-with-the-response-content
      * You can define a "domain" parameter with the current domain of your app.
      *
      * @param bool $delete
      */
-    protected function saveOutput($delete = true)
+    public function saveOutput($delete = true)
```

Comment:
Why changing methods' visibility? If it's only for testing purpose, it's not a great strategy. It's better to change visibility inside tests using reflection.